core/src/main/java/bisq/core/offer/OfferRestrictions.java

```diff
@@ -33,7 +34,7 @@
     private static final Date REQUIRE_TOR_NODE_ADDRESS_V3_DATE = Utilities.getUTCDate(2021, GregorianCalendar.AUGUST, 15);
 
     public static boolean requiresNodeAddressUpdate() {
-        return new Date().after(REQUIRE_TOR_NODE_ADDRESS_V3_DATE);
+        return Config.baseCurrencyNetwork().isRegtest() || new Date().after(REQUIRE_TOR_NODE_ADDRESS_V3_DATE);
```

Comment:
Is this right, always showing the node address update for regtest?

Reply:
Damn - I left my Regtest test code in 😬